### DISCUSSION TITLE
Invoke-PnPBatch results in Invalid file name.

### DISCUSSION THREAD

**Discussion by bhishma** [pnp/powershell#195] [Q&A] [answered]
I'm trying to add list items in a simple site.. but keep getting Invalid file name error:
```
$batch = New-PnPBatch
Add-PnPListItem -List $list -Values @{"Title" ="site1"} -Batch $batch
Add-PnPListItem -List $list -Values @{"Title" ="site2"} -Batch $batch
Add-PnPListItem -List $list -Values @{"Title" ="site3"} -Batch $batch   
$details= Invoke-PnPBatch -Batch $batch -Details

```
results in 

Invalid file name.
The file name you specified could not be used.  It may be the name of an existing file or directory, or you may not have permission to access the file.

I'm using
Manifest   1.2.0      PnP.PowerShell  
Tested with nightly build as well:
Manifest   1.2.7      PnP.PowerShell

Note: it works if I remove the batching.

**Comment by erwinvanhunen** [comment, marked as answer]
I moved this to an issue: https://github.com/pnp/powershell/issues/197
